presto-product-tests/src/main/java/com/facebook/presto/tests/jdbc/JdbcTests.java

```diff
@@ -26,11 +28,7 @@
 import org.testng.annotations.Test;
 
 import java.math.BigDecimal;
-import java.sql.Connection;
-import java.sql.DatabaseMetaData;
-import java.sql.Date;
-import java.sql.SQLException;
-import java.sql.Statement;
+import java.sql.*;
```

Comment:
Don't import * -- make sure you have the airlift codestyle set up in Intellij, so that Intellij doesn't "help" you by doing things like this